python/dgl/data/tu.py

```diff
@@ -21,13 +22,17 @@ class TUDataset(object):
 
     """
 
-    _url = r"https://ls11-www.cs.uni-dortmund.de/people/morris/graphkerneldatasets/{}.zip"
+    _url = r"https://ls11-www.cs.tu-dortmund.de/people/morris/graphkerneldatasets/{}.zip"
 
-    def __init__(self, name, use_pandas=False, hidden_size=10):
+    def __init__(self, name, use_pandas=False, hidden_size=10, n_split=0, split_ratio=None, max_allow_node=1000, cross_valid=False):
```

Comment:
Do not make max_allow_node=1000 as  default. Unless it is specified, otherwise should not remove graphs from dataset.